docs/editor/profiles.md

```diff
@@ -0,0 +1,95 @@
+---
+Order: 15
+Area: editor
+TOCTitle: Profiles
+ContentId: a65efc48-5a2d-4c7d-bd23-03f0393b53f6
+PageTitle: Profiles in Visual Studio Code
+DateApproved: 2/2/2023
+MetaDescription: Expand your development workflow with task integration in Visual Studio Code.
+---
+# Profiles in Visual Studio Code
+
+## Create a Profile
+
+You start with the Default Profile
+
+Can create an Empty Profile or one based on the current Profile.
+
+As you change settings, extension, UI state the profile is updated. Might need to refresh the Profiles view.
+
+You can delete a profile via ....
+
+### Check the current profile
+
+Title bar
+gear hover
+File > Preferences > Profiles (current profile)
+
+### Workspace Associations
+
+When you select create or select an existing profile
+
+## Profile contents
+
+Show Contents brings up the Profiles view
+
+TBD ![alt](https://) that shows the Activity bar icon
+
+### Settings
+
+### Extensions
+
+### UI state
+
+### Keybindings
+
+### Snippets
+
+### User Tasks
+
+## Share Profiles
+
+### Export
+
+As GitHub gist or local file
+
+### Import
+
+## Empty Profiles
+
+## Temporary Profiles
```

Comment:
Do we need to mention Temporary Profiles?
@sandy081 what's the biggest use case?

Reply:
We should explain Temporary Profiles since they will show up at the top of the Command Palette if someone searches on "create profile":

![image](https://user-images.githubusercontent.com/12818376/218604358-ae5f83f5-0506-4627-9efb-f0f00258e820.png)

They don't show up in the File > Prereferences > Profiles > Create Profile menu or gear button fly out but they are in the Command Palette

Reply:
Yeah this can be moved to FaQ section.